components/layout/footer.tsx

```
@@ -0,0 +1,165 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+import { Github, Twitter, Linkedin, Mail } from "lucide-react";
+import { cn } from "@/lib/utils";
```

ESLint

[error] 'cn' is defined but never used.

(@typescript-eslint/no-unused-vars)